app/src/rc3/res/values-pl/strings.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name">Grafik rC3</string>
+    <string name="app_hardcoded_subtitle">27&#8211;30 grudnia 2020, online</string>
```

Comment:
I prefer to use `–` instead of the HTML entity code. I will change this here and in the `ccc36c3` file.